model/PciService.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+namespace oat\qtiItemPci\model;
+
+use oat\qtiItemPci\model\validation\PciModelValidator;
+use oat\qtiItemPci\model\validation\PciValidator;
+use Zend\ServiceManager\ServiceLocatorAwareInterface;
+use Zend\ServiceManager\ServiceLocatorAwareTrait;
+
+class PciService implements ServiceLocatorAwareInterface
```

Comment:
From the look of it, it is more like a PciPackageImportService right ? It is the link between the PciPackageParser and the PciRegistry.
So similarly, I assume we gonna have another PciItemImportService, linking the PciItemParser and the PciRegistry ? Do I understand correct ?